policy/handler_test.go

```diff
@@ -0,0 +1,73 @@
+package policy
+
+import (
+	"testing"
+	"time"
+
+	hclog "github.com/hashicorp/go-hclog"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestHandler_isInCooldown(t *testing.T) {
+	testCases := []struct {
+		inputCooldown  time.Duration
+		inputTimestamp int64
+		inputLastEvent uint64
+		expectedOutput bool
+		name           string
+	}{
+		{
+			inputCooldown:  100 * time.Hour,
+			inputTimestamp: time.Now().UTC().UnixNano(),
+			inputLastEvent: uint64(time.Now().UTC().UnixNano()),
```

Comment:
I don't know much about the details of testing in Go, but it might be better to use [`time.Date()`](https://golang.org/pkg/time/#Date) to have deterministic inputs for tests.

Reply:
Makes sense to be explicit although I would never expect this test to fail. I'll fix it up to use date.